clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -955,6 +956,14 @@ public String toRst() {
             b.append("  * Importance: ");
             b.append(def.importance.toString().toLowerCase(Locale.ROOT));
             b.append("\n\n");
+
+            b.append("  * Dependents: ");
+            for (String dependent : def.dependents)
+            {
+                b.append(dependent);
```

Comment:
The dependent key name should be in backticks. And perhaps comma-delimited rather than tabs.